src/components/FormInput/FormInput.js

```diff
@@ -0,0 +1,37 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+
+export default function FormInput(props) {
+  const [focused, setFocused] = useState(false);
+  const { label, errorMessage, onChange, id, pattern, value, ...inputProps } =
+    props;
+
+  return (
+    <section aria-labelledby="inputlabel">
```

Comment:
This is unnecessary if you connect the label/input correctly I think :)
```suggestion
    <section>
```